### PR TITLE
Change maintainer & like button support

### PR DESCRIPTION
**Pull request by d-r-q**
Change maintainer & like button support

- Author: Aleksey Zhidkov <aleksey.zhidkov@gmail.com>
- Reviewed by: FIXME <FIXME>
- Issue: tiliado/nuvola-app-yandex-music#2

### CODE REVIEW

File: integrate.js [hunk 1]
```diff
@@ -47,17 +47,24 @@ var PlayerAction = Nuvola.PlayerAction;
...
-    
+
     var state = document.readyState;
     if (state === "interactive" || state === "complete")
         this._onPageReady();
     else
         document.addEventListener("DOMContentLoaded", this._onPageReady.bind(this));
+
+    Nuvola.actions.addAction("like", "win", ACTION_LIKE, C_("Action", "Like"),
+        null, null, null, null);
```
Comment (jiri-janousek) [inline integrate.js]: Isn't the Like action a toggle action (on/off)? I mean do you press the button once to like the track and then again to unlike it? For example, the thumbs up action in Google Play and in [Custom Actions](https://tiliado.github.io/nuvolaplayer/development/apps/custom-actions.html) is a toggle action.

**Review by jiri-janousek — COMMENTED** [review]
There is one question regarding the like action.

File: metadata.in.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
     "id": "yandex_music", 
     "name": "Yandex Music", 
-    "maintainer_name": "Jiří Janoušek", 
-    "maintainer_link": "https://github.com/fenryxo", 
+    "maintainer_name": "Aleksey Zhidkov", 
+    "maintainer_link": "https://github.com/aleksey-zhidkov",
```
Comment (jiri-janousek) [inline metadata.in.json]: 404 Not found. Have you renamed your account?

Comment (d-r-q) [reply metadata.in.json]: Yes, my bad, will fix it

File: integrate.js (hunk 1 above)
Comment (d-r-q) [reply integrate.js]: Yes, like it's toggle action. Do you mean that there should be separate actions for like and unlike?

Comment (jiri-janousek) [reply integrate.js]: No, a single action but defined as a toggle action. The key is the last parameter of [Actions.addAction](https://tiliado.github.io/nuvolaplayer/development/apps/api_reference.html#Nuvola.Actions.addAction). It's `null` for simple action and `true/false` for toggle actions:
```js
Nuvola.actions.addAction(
    "playback", "win", ACTION_LIKE, C_("Action", "Like"),
    null, null, null, false);
```

Then you need update the current state in your `WebApp.update()` function with [Nuvola.actions.updateState](https://tiliado.github.io/nuvolaplayer/development/apps/api_reference.html#Nuvola.Actions.updateState).

```js
Nuvola.actions.updateState(ACTION_LIKE, true); // The song is liked
Nuvola.actions.updateState(ACTION_LIKE, false); // The song is not liked
```

Then test them in the developer sidebar. Simple actions are shown as buttons, toggle actions as checkboxes.

custom_actions_enabled

**Comment by d-r-q** [comment]
@fenryxo I fixed maintainer link and define action as toggle action, but I do not see any extra actions in my GUI (shortcut works and I used debug logs to test that it's actually works).

**Comment by jiri-janousek** [comment]
> I do not see any extra actions in my GUI

See https://tiliado.github.io/nuvolaplayer/development/apps/custom-actions.html#show-actions-in-user-interface

**Comment by jiri-janousek** [comment]
:+1: LGTM now. Any more changes to land in 1.5 release? Otherwise, I'll release it at the weekend :tada:

**Comment by d-r-q** [comment]
Thanks, for review:)
I do not plan to implement new features in the near future, so you can release the plugin at the weekend.

**Pull request merged**